Functions/Functions/User/ConnectUserCommentsFunction.cs

```diff
@@ -0,0 +1,89 @@
+using System;
+using System.IO;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Azure.WebJobs;
+using Microsoft.Azure.WebJobs.Extensions.Http;
+using Microsoft.AspNetCore.Http;
+using Microsoft.Extensions.Logging;
+using Newtonsoft.Json;
+using OidcApiAuthorization.Abstractions;
+using OidcApiAuthorization.Models;
+using System.Linq;
+
+namespace SSW.Rules.Functions
+{
+    public class ConnectUserCommentsFunction
+    {
+        private readonly RulesDbContext _dbContext;
+        private readonly IApiAuthorization _apiAuthorization;
+
+        public ConnectUserCommentsFunction(RulesDbContext dbContext, IApiAuthorization apiAuthorization)
+        {
+            _dbContext = dbContext;
+            _apiAuthorization = apiAuthorization;
+        }
+
+        [FunctionName("ConnectUserCommentsFunction")]
```

Comment:
Please comment to explain what this function/class does. I am not clear on what it is supposed to do, a summary comment would help

Reply:
As per Matt W's recommendation please remove the summary comment. Sorry for misleading you! Maybe change the name to ConnectUserToCommentsAccount if you think that is a more descriptive name. Also I'm not sure function is needed in the name since that is implied.